CHANGELOG.md

```diff
@@ -29,6 +29,8 @@ prefixed with the hostname. For example a file that was previously named
 https://github.com/shadow/shadow/pull/2856
 
 * The per-process option `stop_time` has been replaced with `shutdown_time`.
+When set, the signal specified by `shutdown_signal` (a new option) will be sent
+to the process at the specified time.
```

Comment:
Also document that the behaviour has changed (sends a SIGTERM by default rather than a SIGKILL; It's not enough to just rename `stop_time` with `shutdown_time` to keep the same behaviour)?